ioctl/newcmd/config/config_reset_test.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"os"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-core/test/mock/mock_ioctlclient"
+	"github.com/iotexproject/iotex-proto/golang/iotexapi/mock_iotexapi"
+)
+
+func TestConfigReset(t *testing.T) {
+	require := require.New(t)
+	ctrl := gomock.NewController(t)
+	client := mock_ioctlclient.NewMockClient(ctrl)
+	client.EXPECT().Config().Return(config.Config{})
+	apiServiceClient := mock_iotexapi.NewMockAPIServiceClient(ctrl)
+
+	client.EXPECT().APIServiceClient().Return(apiServiceClient, nil).Times(1)
+
+	cmd := NewConfigReset(client)
+	result, err := util.ExecuteCmd(cmd, "reset")
```

Comment:
1. the failure case with returning error is not simulated or covered
2. unit test for `config.reset` should be added.

Reply:
I have added a failure case for the command and a unit test for the reset. I had to pass in the config file name into the command to be able to simulate an error, let me know what you think. Cheers